easybuild/easyblocks/o/openblas.py

```diff
@@ -18,3 +18,18 @@ def build_step(self):
             self.cfg['buildopts'] += ' TARGET=POWER8'
 
         super(EB_OpenBLAS, self).build_step()
+
+    def configure_step(self):
+        """ nothing to do for OpenBLAS """
+        pass
+
+    def sanity_check_step(self):
+        """ Custom sanity check for OpenBLAS """
+        custom_paths = {
+            'files': ['include/cblas.h', 'include/f77blas.h', 'include/lapacke_config.h', 'include/lapacke.h',
+                      'include/lapacke_mangling.h', 'include/lapacke_utils.h', 'include/openblas_config.h',
+                      'lib/libopenblas.a', 'lib/libopenblas.%s' % get_shared_lib_ext()],
+
+            'dirs': [''],
```

Comment:
@edmondac Please change this to `[]` (check for a single directory with an empty name is silly)